assets/constants.ts

```diff
@@ -17,6 +17,8 @@ export = {
   // Whether to allow custom event reporting to Google Analytics.
   "CAN_SEND_ANALYTICS_EVENTS": false,
 
+  "CLASSROOM_STATUS_STAGING": "staging",
```

Comment:
I'm not sure "CLASSROOM_STATUS" is valid for this -- a "status" sounds like something that can be applied to any classroom. Perhaps DUMMY_CLASSROOM_NAME and provide some explanation of what this is in the comments.